specutils/tests/test_loaders.py

```diff
@@ -707,11 +707,10 @@ def test_tabular_fits_compressed(compress, tmpdir):
 
 
 @pytest.mark.parametrize("spectral_axis",
-                         ['wavelength', 'frequency', 'energy', 'wavenumber'])
+                         ['WAVE', 'FREQ', 'ENER', 'WAVN'])
```

Comment:
Just some minor context: if I recall, the spectral axis names were meant to correspond with the class properties and not the FITS naming standard. Either way, it's not an issue, but figured the history of the usage might add some context.